proto/api/read_state.proto

```diff
@@ -29,6 +29,13 @@ message ReadStakingDataMethod {
 		CANDIDATE_BY_ADDRESS = 7;
 		TOTAL_STAKING_AMOUNT = 8;
 		BUCKETS_COUNT = 9;
+		COMPOSITE_BUCKETS = 10;
```

Comment:
let's start from 20, in case anything needed on native's side

Reply:
fixed